.ci/snap_publish.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -euo pipefail
+
+SNAPCRAFT_BUILD_ENVIRONMENT_CPU=$(nproc)
+export SNAPCRAFT_BUILD_ENVIRONMENT_CPU
+SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY=16G
+export SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY
```

Comment:
```
export SNAPCRAFT_BUILD_ENVIRONMENT_CPU=$(nproc)
export SNAPCRAFT_BUILD_ENVIRONMENT_MEMORY=16G
```
?

Reply:
And the same for `THREADS` and `ROOT`

Reply:
Shell check suggestion. Export+assign+command makes it fail silently.

Reply:
Woops.. that means memory could have gone on a single line.

Reply:
I had no idea.

Reply:
Yeah, me neither!